### DISCUSSION TITLE
Underscore symbol raises a parser error

### DISCUSSION THREAD

**Discussion by axlnur** [FabCx/TradingView-Webhook-Bot#73] [General]
Hello!

First of all thank you for your Webhook Bot! It works perfect and was easy to setup!

The parser receives an error when the alert text contains an Underscore symbol [here]--> _ <--[here].

Example:
```
{
 "msg":"Test_alert", "key":"fuVAtKh2gm"
}
```

Result:

```
0|tvalertbot_m  | [X] 2021-08-13 13:59:38 Error:
0|tvalertbot_m  | > Can't parse entities: can't find end of the entity starting at byte offset 4

```

How can I fix this?

Thanks,
-DrMavrodi

**Comment by fabston** [comment]
Hi there,

Try escaping it:

```
{
 "msg":"Test\_alert", "key":"fuVAtKh2gm"
}
```

**Reply by axlnur** [reply]
Unfortunately, escaping won't help us in the next case:

We get the stock ticker and the asset ticker using variables: {{exchange}}:{{ticker}}

Example:
`{
 "msg":"{{exchange}}:{{ticker}}", "key":"fuVAtKh2gm"
}
`

If you set an alert here (XETR:DHER), it will throw an error when the alert is triggered because the stock ticker itself contains the Underscore symbol (XETR_DLY:DHER). :(

**Reply by fabston** [reply]
Gotcha. I just updated the code. Please pull it and let me know if it fixed the issue.

**Reply by axlnur** [reply]
Thank you very much for your feedback!

I used the command now:
`git clone https://github.com/fabston/TradingView-Webhook-Bot.git
`
But the bug is still active. :(

`2|tvalertbot_m  | Can't parse entities: can't find end of the entity starting at byte offset 41
`
Or have I installed a past version of the Webhook Bot using the command above now?

**Reply by fabston** [reply]
You could’ve also just typed `git pull` inside your working dir. 

I’ll have to test it on my end then. Was hoping the commit would’ve fixed the issue. Not sure when I’ll find time for that though.
